INSTALLATION.md

```diff
@@ -0,0 +1,132 @@
+
+Accedere allo Storage account e configurare il CORS, cliccando su Resource Sharing e impostando im blob service l'origine (l'url del sito statico) e gli allowed methods (GET e OPTIONS). Dopo avere inserito star (*) in Allowed Header e in Exposed Header, impostare un Max Age di 200.
+
+Successivamente salvare la configurazione CORS e procedere con il caricamento dei file statici.
+
+Nella folder della CustomUI sono presenti tutti i file della UI che dovranno essere caricati nello storage. All'interno di questi file sono presenti dei placeholder che fanno riferimento alle proprietà CustomUiBlobStorageUrl e MetadatasBlobStorageUrl del file [appsetting.json](https://github.com/microsoft/SPID-and-Digital-Identity-Enabler/blob/main/AAD%20B2C/CustomPolicies/appsettings.json).
+
+Trovando i placeholder dentro i files della customUI e sostituendo il valore con l'url rispettivo i file sono pronti per essere caricati nello storage nella cartella customUI dentro $web.
+
+### Caricamento dei Metadata dentro lo Storage Account ###
+Nei file dei metadati dobbiamo sostituire gli endpoint con quelli dello spidproxy e dobbiamo generare un certificato dello spidproxy seguendo le norme AGID. 
+
+Per creare il certifcato dello spidproxy dobbiamo utilizzare il tool [spid compliant certificate](https://github.com/italia/spid-compliant-certificates). Le regole per creare il certificato sono indicate a questo link> [Avviso SPID n.29 v3](https://www.agid.gov.it/sites/default/files/repository_files/spid-avviso-n29v3-specifiche_sp_pubblici_e_privati_0.pdf). E' possibile creare un certificato per i soggetti pubblici, mentre per i privati si crea un CSR e lo si spedisce ad AGID che a sua volta rimanderà un certificato indietro.
+Copiare il file public.env.example e rinominarlo in docker.env. Succesivamente modificarne i valori all'interno indicando i valori del soggetto pubblico. Le informazioni sono ricercabili sul sito di AGID. 
+
+- **COMMON_NAME**: la denominazione che valorizza l'estensione organizationName, senza acronimi.
+
+- **DAYS**: durata in giorni.
+
+- **ENTITY_ID**: uri contenente identificativo che inseriamo nel metadata che inviamo ad agid.
+
+- **KEY_LEN**: lasciare 3072.
+
+- **LOCALITY_NAME**: indicare la città dell'ente.
+
+- **MD_ALG**: sha256.
+
+- **ORGANIZATION_IDENTIFIER**: Codice IPA che si trova dall'indica PA di AGID. Attenzione al limite dei 64 caratteri. In caso di lunghezza maggiore mettere l'acronimo o qualcosa di simile
+
+- **ORGANIZATION_NAME**: nome completo come indicato nei pubblici registri. 
+
+- **SPID_SECTOR**: public
+
+Dopo avere settato questi valori seguire la [guida](https://github.com/italia/spid-compliant-certificates#private-key-csr-and-self-signed-certificate-for-public-sector-with-docker) eseguendo lo script gencert-with-docker.sh.
```

Comment:
rimanderei direttamente al readme sul loro github, così evitiamo di dover mantenere queste info nel caso in cui dovessero cambiare nel tempo